tool/requote.js

```diff
@@ -19,6 +19,7 @@ const { maybeCreateReadStream, readAllLines } = require('./lib/argutils');
 const StreamUtils = require('../lib/stream-utils');
 
 const ENTITY_MATCH_REGEX = /^([A-Z].*)_[0-9]+$/;
+const NUMBER_MATCH_REGEX = /^([0-9|\u0660-\u0669]+)$/;
```

Comment:
Do you really need arabic numerals in the sentence? I don't think we want non-ASCII in ThingTalk syntax (outside of `"` marks). So you'd need to replace the numbers in the sentence using arabic numerals, and replace the code with regular numbers. Tricky, I suppose.

Reply:
true, I will omit those characters since I'm using English digits for Farsi anyways.